src/main/java/cloud/orbit/actors/extensions/metrics/dropwizard/DatadogReporterConfig.java

```diff
@@ -77,7 +77,13 @@ else if ("udp".equalsIgnoreCase(mode))
         DatadogReporter datadogReporter = null;
         try
         {
-            datadogReporter = DatadogReporter.forRegistry(registry).withHost(InetAddress.getLocalHost().getCanonicalHostName()).withPrefix(this.getPrefix()).withTransport(transport).build();
+            datadogReporter = DatadogReporter.forRegistry(registry)
+                .convertRatesTo(getRateTimeUnit())
```

Comment:
The convertRatesTo and convertDurationsTo was missing for the DatadogReporterConfig.